avocado/core/nrunner.py

```diff
@@ -318,7 +318,8 @@ class BaseRunner:
     def __init__(self, runnable):
         self.runnable = runnable
 
-    def prepare_status(self, status_type, additional_info=None):
+    @staticmethod
+    def prepare_status(status_type, additional_info=None):
```

Comment:
Maybe here we should also consider it as an abstract method as this is the BaseRunner class.

Reply:
In fact, maybe for ``BaseRunner`` it would be worth making it an abstract class. At least it seems a bit clearer to me.

Reply:
Do you want to send a PR proposing that? I can do that if you prefer.

Reply:
Sure! :) I will disable the check here as well.

Reply:
> Sure! :) I will disable the check here as well.

Hi @tiagohonorato, I saw that you already did the force push, but without this change. Will you paining to disable the check here or did I miss something?

Reply:
Hi @richtja, I will send a PR proposing ``BaseRunner`` as an abstract class and ``run`` as an abstractmethod. This way we will not need it to be a staticmethod. Let me know if this answers your question.